cmd/mattermost/commands/server.go

```diff
@@ -34,16 +36,42 @@ func init() {
 	RootCmd.RunE = serverCmdF
 }
 
+func loadCustomDefaults() (*model.Config, error) {
+	var customDefaults *model.Config
+	customDefaultsPath := os.Getenv("MM_CUSTOMDEFAULTS")
+
+	if customDefaultsPath != "" {
```

Comment:
Nit: we can return early here to reduce one level of indentation below

Reply:
Addressed in 02e8146d2